modules/manage/partials/monitor-redpanda.adoc

```diff
@@ -281,11 +281,15 @@ To detect unexpected idling, you can query the rate of change as a percentage of
 rate(redpanda_cpu_busy_seconds_total[5m])
 ----
 
-[NOTE]
+[TIP]
 ====
-While CPU utilization at the host-level can typically appear high (for example, 99-100% utilization) when I/O events like message arrival occur, the actual Redpanda process utilization is likely low and can be monitored with `redpanda_cpu_busy_seconds_total`.
+While CPU utilization at the host-level can typically appear high (for example, 99-100% utilization) when I/O events like message arrival occur, the actual Redpanda process utilization is likely low, in contrast to what system-level metrics might indicate such as those provided by the `top` command. 
+
+This high host-level CPU utilization happens because Redpanda uses Seastar, which runs event loops on every core (also referred to as a _reactor_), constantly polling for the next task. This process never blocks and will increment clock ticks. It doesn't necessarily mean that Redpanda is busy.
+
+Use `redpanda_cpu_busy_seconds_total` (public metrics) or xref:reference:internal-metrics-reference.adoc#vectorized_reactor_utilization[`vectorized_reactor_utilization`] (internal metrics) to monitor the actual Redpanda CPU utilization. When these particular metrics approach 100% utilization, you will then also start to see higher produce and consume <<latency,latency>>.
```

Comment:
Recommend adding a hyperlink to the `redpanda_cpu_busy_seconds_total` metric. Also, you might consider rephrasing this. The CPU Busy Seconds metric is an incrementing count of busy seconds, so it might be best monitored over a time slice to see heavy utilization, while the reactor utilization is more in line with showing a % utilizaion.

Reply:
@greg-szrama I left the link out since we have it earlier in the section but on second thought, no harm in adding it here again. Yes, it should specify the "calculation" for utilization and not the count itself... I'll find a way to rephrase. Thanks!